gno.land/cmd/gnoland/testdata/addpkg_outofgas.txtar

```diff
@@ -0,0 +1,57 @@
+# ensure users get proper out of gas errors when they add packages
+
+# start a new node
+gnoland start
+
+# add foo package
+gnokey maketx addpkg -pkgdir $WORK/foo -pkgpath gno.land/r/foo -gas-fee 1000000ugnot -gas-wanted 211483 -broadcast -chainid=tendermint_test test1
```

Comment:
```suggestion
gnokey maketx addpkg -pkgdir $WORK/foo -pkgpath gno.land/r/foo -gas-fee 1000000ugnot -gas-wanted 50000 -broadcast -chainid=tendermint_test test1
```

like the other comment, let's maybe not make this a test against specific gas results

Reply:
Same reason as above:

The chain consumes gas at multiple points outside the VM before reaching the VM. Even inside the VM, a transaction may run out of gas during preprocessing or runtime. To precisely replicate and verify that the bug is fixed in the preprocessing stage, we need to specify an exact gas amount in the integration test.

Reply:
Same here, this command is not supposed fail; in fact it's not prefixed with `!`.

Reply:
@thehowl I looked at the wrong place. Yes, this should not fail.  

How about we set the gas limit to 50% more than the amount used? This way, if the test fails due to running out of gas, we can at least be alerted and identify the root cause.